src/main/webapp/css/landing.css

```diff
@@ -18,6 +18,16 @@
     margin-bottom: 10px;
 }
 
+.user-favorite {
+    font-size: 30px;
+}  
+
+.dashboard {
+    text-align: center;
+    font-size: 80px;
+}
+
+
```

Comment:
Please keep the blank lines usage consistent throughout the file. I see we have single blank lines above.